homeassistant/components/onewire/sensor.py

```diff
@@ -184,11 +184,11 @@ class OneWireSensorEntityDescription(OneWireEntityDescription, SensorEntityDescr
             state_class=STATE_CLASS_MEASUREMENT,
         ),
         OneWireSensorEntityDescription(
-            key="IAD",
+            key="vis",
             device_class=DEVICE_CLASS_CURRENT,
             entity_registry_enabled_default=False,
-            name="Current",
-            native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
```

Comment:
Maybe just `vis` not "current vis"?

Reply:
@epenet If the IAD is a switch that tells us to read or not read the vis, isn't that something that would go into the new entity class of CONFIG, ie, so it doesn't show up on the main page, just interior to the integration?

Reply:
As pointed out by @garbled1 please add 
```python
entity_category=ENTITY_CATEGORY_CONFIG,
```

Reply:
to be clear here, you can just add the category like that, as we have 2 other PR's that will be handling the tests for it.  (you inspired me)